tests/test_linting.py

```diff
@@ -1,5 +1,11 @@
 from goose.toolkit.lint import lint_toolkits
 
+from goose.toolkit.lint import lint_providers
+
 
 def test_lint_toolkits():
     lint_toolkits()
+
+
+def test_lint_providers():
+    lint_providers()
```

Comment:
I guess you followed the existing implementation pattern. 

I am not sure whether `lint_providers` function is used anywhere else.   If it does not, Iwe can move the implementation of `lint_providers` into this test_linting.py as the implementation is just to check the doc string is correct.  I feel it will be easier for people to read and understand

Reply:
@lukealvoeiro can describe it better, but the reason behind this is that `lint_providers` could be used in tests for the `goose-plugins-block` internal library. Currently, we only have one internal provider, but if there are more in the future, they should follow the same structure. Do you think we should reconsider this logic?

Reply:
Yup, we want this function to be exported so we can lint this in any open source or internal plugin repos as well.